CMakeLists.txt

```diff
@@ -28,6 +28,11 @@ add_custom_target(run
     COMMAND ${CMAKE_SOURCE_DIR}/Meta/run.sh
     USES_TERMINAL
 )
+add_custom_target(all-and-run
+    DEPENDS all install qemu-image run
```

Comment:
doesn't the `DEPENDS ... run` make it run run.sh twice?

Reply:
I tried this and it made the image before finishing compiling.

Reply:
Yup copy pasta bug, thanks!